PendingReleaseNotes.md

```diff
@@ -15,6 +15,10 @@ v1.7...
   - See [the design](https://github.com/rook/rook/blob/master/design/ceph/resource-dependencies.md)
     for detailed information.
 
+- OSDs
```

Comment:
I added this to my notes for the 1.6.7 release, we really only need to update this file for minor releases.

Reply:
I added this to merge into master for after #8200